solr/core/src/test/org/apache/solr/update/processor/CategoryRoutedAliasUpdateProcessorTest.java

```diff
@@ -233,6 +233,54 @@ public void testMissingRequiredParams() throws Exception {
     assertTrue("Create Alias should fail since not all required params were supplied", e.getMessage().contains("Not all required params were supplied"));
   }
 
+  @Slow
+  @Test
+  public void testMustMatch() throws Exception {
+    String configName = getSaferTestName();
+    createConfigSet(configName);
+    final String regexEnding = "_solr";
```

Comment:
This test would be clearer if the regex were expressed as a single string.

Reply:
On it.